src/core.js

```
@@ -157,7 +157,8 @@ export default class CoreChart extends BaseChart {
 
   // 传入简化后的icharts-option
   setSimpleOption(ChartClass, option, plugins = {}, isInit = true) {
-    let iChartOption = cloneDeep(option);
+    let iChartOption = {};
+    merge(iChartOption, option);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)